### DISCUSSION TITLE
from api to create shorturl very slow!!

### DISCUSSION THREAD

**Discussion by Marco1221** [YOURLS/YOURLS#3100] [Q&A] [answered]
I Deactivate my all plugins for try,
my request :
https://y.*****.com/yourls-api.php?title=test&signature=79******89&action=shorturl&url=https://google.com
resp:

Status: 200OK
Time: **178 ms**
Size:600 B

who can help me, thanks!!

**Comment by dgw** [comment, marked as answer]
178ms is "slow"? For a full round trip to the server, connecting to the database, all that's involved in processing this API request? Doesn't sound slow.

**Reply by ozh** [reply]
Agreed. Anyway, if too slow by your standards, there's nothing we can do. It depends on your hosting and server configuration.
